pom.xml

```diff
@@ -127,6 +129,21 @@
                 <artifactId>jjwt-jackson</artifactId>
                 <version>${jjwt.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.apache.camel.quarkus</groupId>
+                <artifactId>camel-quarkus-kamelet</artifactId>
+                <version>${quarkus.camel.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.apache.camel.quarkus</groupId>
+                <artifactId>camel-quarkus-timer</artifactId>
+                <version>${quarkus.camel.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.apache.camel.quarkus</groupId>
+                <artifactId>camel-quarkus-direct</artifactId>
+                <version>${quarkus.camel.version}</version>
```

Comment:
We should add the bom dependency instead of individual dependencies: https://mvnrepository.com/artifact/org.apache.camel.quarkus/camel-quarkus-bom